src/components/TextInput.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Field } from 'redux-form';
+import { TextField } from 'redux-form-material-ui';
```

Comment:
This should be `from 'material-ui'`. The goal is to drop the dependency on `redux-form-material-ui`.